dev/io.openliberty.jakarta.expressionLanguage.4.0/bnd.bnd

```diff
@@ -24,7 +24,8 @@ Export-Package: \
        uses:="jakarta.el"
 
 Include-Resource: \
-    jakarta=resources/jakarta
+    jakarta=resources/jakarta, \
+    @${repo;org.apache.tomcat:tomcat-el-api;10.0.18;EXACT}!/!(META-INF/maven/*|module-info.class)
```

Comment:
will this get rid of overlay?  Does the order need to get flipped?

Reply:
Does not appear to but probably doesn't hurt to flip it anyway.